doc_source/troubleshooting.md

```diff
@@ -76,9 +76,141 @@ If your managed node group encounters a health issue, Amazon EKS returns an erro
 + **NodeCreationFailure**: Your launched instances are unable to register with your Amazon EKS cluster\. Common causes of this failure are insufficient [node IAM role](worker_node_IAM_role.md) permissions or lack of outbound internet access for the nodes\. Your nodes must be able to access the internet using a public IP address to function properly\. For more information, see [VPC IP addressing](network_reqs.md#vpc-cidr)\. Your nodes must also have ports open to the internet\. For more information, see [Amazon EKS security group considerations](sec-group-reqs.md)\.
 + **InstanceLimitExceeded**: Your AWS account is unable to launch any more instances of the specified instance type\. You may be able to request an Amazon EC2 instance limit increase to recover\.
 + **InsufficientFreeAddresses**: One or more of the subnets associated with your managed node group does not have enough available IP addresses for new nodes\.
-+ **AccessDenied**: Amazon EKS or one or more of your managed nodes is unable to communicate with your cluster API server\.
++ **AccessDenied**: Amazon EKS or one or more of your managed nodes is unable to communicate with your cluster API server\. See below for more details on how to fix this issue\.
 + **InternalFailure**: These errors are usually caused by an Amazon EKS server\-side issue\.
 
+### Fixing AccessDenied errors for managed node groups
+
+The most common cause of `AccessDenied` errors when performing operations on managed node groups is missing the `eks:node-manager` ClusterRole and/or ClusterRoleBinding. Amazon EKS sets up these resources in your cluster as part of onboarding with managed node groups, and these are required for managing the node groups.
+
+You can verify the cause and resolve the issue with a few `kubectl` commands.
+
+**Expected ClusterRole and ClusterRoleBinding**
+
+The ClusterRole and ClusterRoleBinding are subject to change over time, but they should look similar to the following:
```

Comment:
Instead, should we put it as, what "should" be there? Saying, "are subject to change over time" might be a little confusing, since the permissions to these roles pretty scooped down

Reply:
We can. My only concern is that at some point in time, we could theoretically lock them down further or need to add additional permissions. I don't know how realistic that is and maybe it will be less confusing to not go into that, but I think it's worth discussing what makes sense.